pwndbg/commands/nearpc.py

```diff
@@ -97,6 +99,10 @@ def nearpc(pc=None, lines=None, to_string=False, emulate=False):
 
         line   = ' '.join((prefix, "%#x" % i.address, s or '', asm))
 
+        # Highlight the current line if the config is enabled
```

Comment:
The screenshots that I posted were with your most recent suggested changes.

If you'd like to modify your PR to highlight as shown in that screenshot (i.e. prefix, address, and symbol name), that's fine with me.